### PR TITLE
Reproduce chainID issue for CoinType

### PR DESCRIPTION
**Pull request by rsrbk**

### CODE REVIEW

File: swift/Tests/Blockchains/CosmosTests.swift [hunk 1]
```diff
@@ -8,6 +8,11 @@ import XCTest
 import WalletCore
 
 class CosmosAddressTests: XCTestCase {
+
+    func testChainID() {
+        XCTAssertEqual(CoinType.cosmos.chainId, "cosmoshub-4") // CoinType.cosmos.chainId returns ""
```
Comment (miloserdow) [inline swift/Tests/Blockchains/CosmosTests.swift:13]: Perhaps remove this comment? It doesn't make any sense to me :)

Comment (miloserdow) [reply swift/Tests/Blockchains/CosmosTests.swift:13]: I mean it should fail if it returns "", right?

**Comment by hewigovens** [comment]
We can add current cosmos hub mainnet chain id to registry.json like [this](https://github.com/trustwallet/wallet-core/blob/master/registry.json#L2287, but it might be outdated time to time due to network upgrade, client should not rely on this value in general

**Comment by catenocrypt** [comment]
Correct. The chainId in the registry is just a default value, should be treated as informational only. In SigningInput, chainId must be specified every time, by the client.
Is there any change/improvement you suggest?

**Comment by rsrbk** [comment]
@hewigovens where would you suggest to get `chainID` values from? We could hardcode it on clients, but that would essentially cause similar problem, only that in case of network upgrade, the values need to be updated on all the clients.

**Comment by ArtemGoryunov** [comment]
`chain_id` can be get from the node via call `/cosmos/base/tendermint/v1beta1/node_info`

**Comment by hewigovens** [comment]
Let's add current chain ids in https://github.com/trustwallet/wallet-core/pull/2558

**Pull request closed without merging**